thoth-repo-init/base/argo-workflows/thoth-repo-init.yaml

```
@@ -0,0 +1,58 @@
+---
+apiVersion: argoproj.io/v1alpha1
+kind: WorkflowTemplate
+metadata:
+  name: thoth-repo-init
+spec:
+  templates:
+    - name: thoth-repo-init
+      resubmitPendingPods: true
+      inputs:
+        parameters:
+          - name: "project-url"
+      container:
+        image: image-registry.openshift-image-registry.svc:5000/thoth-test-core/workflow-helpers:latest
```

```suggestion
        image: workflow-helpers:latest
```

we can use this, as we would be patching this with kustomize file.